src/Handler/Home.hs

```diff
@@ -30,12 +30,8 @@ getHomeR :: Handler Html
 getHomeR = track "Handler.Snapshots.getAllSnapshotsR" $ do
     cacheSeconds $ 60 * 60
     now' <- getCurrentTime
-    currentPageMay <- lookupGetParam "page"
```

Comment:
IIUC, this isn't a good idea. We're already having trouble with long database queries, isn't this disabling paging?

Reply:
I may be wrong, this is just for dropping the redundant https://www.stackage.org/?page=2 "feature".

Reply:
Doh, my mistake, sorry. Thanks for clarifying.